src/components/AssetDescriptionComponent.jsx

```diff
@@ -24,82 +26,66 @@ const AssetDescriptionComponent = props => (
       <Grid.Column>
-        <Confirm
-          content="Are you sure you want to make this change?"
-          open={props.open}
-          cancelButton={
-            <ButtonComponent
-              className="cancel"
-              buttonName="Cancel"
-              handleClick={props.handleCancel}
-            />
-          }
-          confirmButton={
-            <ButtonComponent
-              className="save"
-              buttonName="Save"
-              color="primary"
-              buttonState={props.buttonState}
-              handleClick={props.handleConfirm}
-            />
-          }
-        />
-        {(!isEmpty(values(props.assignedUser))) ?
-          <div id="allocate-asset">
-            <Header as="h3" content="Assigned To:" />
-            <div
-              id="email"
-              className="asset-specs"
-            >
-              {props.assignedUser.email}
+        {
+          (!isEmpty(values(props.assignedUser)))
+          ?
+            <div id="allocate-asset">
+              <Header as="h3" content="Assigned To:" />
+              <div
+                id="email"
+                className="asset-specs"
+              >
+                {props.assignedUser.email}
+              </div>
             </div>
-            <br />
-            <ButtonComponent
-              buttonName="Unassign Asset"
-              customCss="unassign-asset"
-              handleClick={props.show}
-              color="primary"
-            />
-          </div>
           :
-          <div id="allocate-asset">
-            <Header as="h3" content="Assign this asset to:" />
-            <DropdownComponent
-              customClass="form-dropdown"
-              label="Assign this asset to:"
-              placeHolder="Select Andela Email To Assign This Asset"
-              name="assign-user"
-              search
-              value={props.selectedUserId}
-              onChange={props.onSelectUserEmail}
-              options={userEmailsOptions(props.users)}
-            />
-            <br />
-            <ButtonComponent
-              buttonName="Assign Asset"
-              customCss="assign-asset"
-              handleClick={props.show}
-              disabledState={props.assignAssetButtonState}
-              color="primary"
-            />
-          </div>
+            <div id="allocate-asset">
+              <Header as="h3" content="Assign this asset to:" />
+              <DropdownComponent
+                customClass="form-dropdown"
+                label="Assign this asset to:"
+                placeHolder="Select Andela Email To Assign This Asset"
+                name="assign-user"
+                search
+                value={props.selectedUserId}
+                onChange={props.onSelectUserEmail}
+                options={userEmailsOptions(props.users)}
+              />
+            </div>
         }
+
+        <ModalComponent
+          trigger={
+            (!isEmpty(values(props.assignedUser)))
```

Comment:
Same here. The ternary operator reduces readability and comprehesnion of the code.   Create a prop that determines the button props and use one `ButtonComponent`. I.e.
```
const AssetDescriptionComponent = props => {
  let triggerProps = {
    buttonName: 'Assign Asset',
    customCss:  'assign-asset',
    disabledState: props.assignAssetButtonState,
    color: primary
  };
  if (!isEmpty(values(props.assignedUser))) {
    triggerProps = {
      ...triggerProps,
      buttonName:  'Unassign Asset',
      customCss: 'unassign-asset',
      disabledState: false
    };
  }

  return (
    ....
    <ModalComponent
      trigger={<ButtonComponent {...triggerProps} />}
      modalTitle="Confirm Action"
    >
      <ConfirmAction {...props} />
    </ModalComponent>
    ....
  );
};
```

If you are not sure if the ternary operator is appropriate for the use case, try reading what you have written aloud in english sentences (i.e. interpreting what have been written and reading the interpretation in english). If it gets difficult to comprehend, then another person who reads the code will have an even harder time understing it.